pixelsort/pixelsort.py

```diff
@@ -0,0 +1,105 @@
+import logging
+from PIL import Image
+
+from pixelsort.util import crop_to
+from pixelsort.sorter import sort_image
+from pixelsort.constants import defaults, black_pixel
+from pixelsort.interval import choices as interval_choices
+from pixelsort.sorting import choices as sorting_choices
+
+
+def pixelsort(
+        image,
+        mask_image=None,
+        interval_image=None,
+        randomness: float = defaults["randomness"],
+        clength: int = defaults["clength"],
+        sorting_function: str = defaults["sorting_function"],
+        interval_function: str = defaults["interval_function"],
+        lower_threshold: float = defaults["lower_threshold"],
+        upper_threshold: float = defaults["upper_threshold"],
```

Comment:
@ethanhs Is there also a cleaner way to do what I did here? I defined some default constants so that the imported function and argparse would behave the same way.

Reply:
Constants in Python are traditionally `UPPER_CASE`. I would just put these values in the `pixelsort.constants` file and access them via `constants.SORTING_FUNCTION` or such.